src/schema/resolvers/user-resolver.js

```diff
@@ -0,0 +1,12 @@
+import * as user from '../../controllers/graphql/user-controller'
+
+export default {
+  Query: {
+    users: (parent, args, { db }) => db.user.findAll(),
+    user: (parent, { id }, { db }) => db.user.findByPk(id)
+  },
+  Mutation: {
+    createUser: (parent, args) => user.createUser(args),
+    deleteUser: (parent, args) => user.deleteUser(args)
+  }
```

Comment:
This file should only be a mapping to tell Apollo what functions to execute for each field of your schema. Change it so that the functions in the controller receive the arguments from Apollo directly:
```javascript
// user-resolver.js
  Query: {
    users: user.getAllUsers,
    user: user.getUserById
  },
  Mutation: {
    createUser: user.createUser,
    deleteUser: user.deleteUser
  }

// models/user.js
export const getAllUsers = (_, args, { models }) => models.user.findAll()
export const getUserById = (_, { id }, { models }) => models.user.findByPk(id)
// ...etc
```